From e49c0b19cfcdfd55a51ca81d9794eeb32a1b1306 Mon Sep 17 00:00:00 2001
From: Richard Biener <rguenther@suse.de>
Date: Wed, 24 Apr 2019 12:54:19 +0800
Subject: [PATCH 3/4] Fix isl 2.0 build error

 -graphite.h: Include isl/id.h and isl/space.h to allow build
  with ISL 0.20.tmp
---
 gcc/ChangeLog  | 6 ++++++
 gcc/graphite.h | 2 ++
 2 files changed, 8 insertions(+)

diff --git a/gcc/ChangeLog b/gcc/ChangeLog
index 42ff8e4b4a9..6b536a4e914 100644
--- a/gcc/ChangeLog
+++ b/gcc/ChangeLog
@@ -1,3 +1,9 @@
+2018-08-01  Richard Biener  <rguenther@suse.de>
+
+	PR bootstrap/86724
+	* graphite.h: Include isl/id.h and isl/space.h to allow build
+	with ISL 0.20.
+
 2017-05-12  Kito Cheng  <kito.cheng@gmail.com>
 
 	* config/riscv/riscv.c (riscv_legitimize_move): Handle
diff --git a/gcc/graphite.h b/gcc/graphite.h
index d22c4f2693f..e69edf2ae15 100644
--- a/gcc/graphite.h
+++ b/gcc/graphite.h
@@ -37,6 +37,8 @@ along with GCC; see the file COPYING3.  If not see
 #include <isl/schedule.h>
 #include <isl/ast_build.h>
 #include <isl/schedule_node.h>
+#include <isl/id.h>
+#include <isl/space.h>
 
 typedef struct poly_dr *poly_dr_p;
 
-- 
2.17.1

